### DISCUSSION TITLE
how to obtain the length of list of address?

### DISCUSSION THREAD

**Discussion by argasec** [vyperlang/vyper#2346] [Q&A]
Im trying to compile this code

def balanceOfBatch(accounts: address[50], ids: uint256[50]) -> uint256[100]:
    assert len(accounts) == len(ids), "ERC1155: accounts and ids length mismatch"

but tells me that len only function for bytes and string types

**Comment by fubuloubu** [comment]
I made an issue for this, but honestlly you don't need to do that check. Both arrays are "static arrays" meaning they cannot be less than `50` in length, so the length check is kind of unnecessary.

Still, it should be a thing you can do: https://github.com/vyperlang/vyper/issues/2347
